tests/test_monotonic_constraints.py

```
@@ -0,0 +1,1289 @@
+            output_dim=self.output_dim,
+            tree_struct=tree_struct,
+            optimizers=optimizer,
+            params=params,
+            verbose=0,
+            device='cpu'
+        )
+        
+        model.set_bias_from_targets(self.y)
+        
+        # Train using step() to apply constraints per tree
+        for epoch in range(self.n_epochs):
```

Ruff (0.14.14)

[warning] Loop control variable `epoch` not used within loop body

Rename unused `epoch` to `_epoch`

(B007)